.gitignore

```diff
@@ -1,4 +1,6 @@
+decred/build
```

Comment:
We want to disregard the `tinywallet` build dir too, so just `build/` should work. (Same goes for `decred/dist/` below, my mistake.)